src/bokeh/server/server.py

```diff
@@ -18,6 +18,8 @@
     :class:`~bokeh.application.application.Application` instances, and will
     automatically create and coordinate the lower level Tornado components.
 
+See :ref:`userguide_server_introduction` for information on server.
```

Comment:
This is a great place for a link, just a small suggested edit:

```suggestion
See :ref:`userguide_server_introduction` for general information on the Bokeh server.
```